sky/engine/core/dart/painting.dart

```diff
@@ -162,6 +162,13 @@ class Path extends NativeFieldWrapperClass2 {
   /// reset to the origin.
   void reset() native "Path_reset";
 
+  /// Tests to see if the path is within the path. (That is, whether
```

Comment:
Wow even after your comment I still read this sentence a half dozen times before realising the problem.